README.md

```diff
@@ -1,43 +1,153 @@
-# Project 
-Exercise tracker
+<!-- PROJECT -->
+  <p align="center">
+   Phisical Fitness
+    <br />
+        <img src="/assets/images/github.png"
+        alt="GitHub"
+             height="80"></a>
+    <br />
+    ![Github](https://github.com/sisay2405/phiscal-fitness-app/blob/feature/readme/src/assets/images/github.png)
```

Comment:
Please correct link to point to GitHub itself rather than the image, and correct so the text of "GitHub" shows as link text.